### ISSUE TITLE
Finals Code Editor

### ISSUE THREAD

**Issue by ca-leb** [nus-cs2030/2122-s2#571]
Hi, can i check if anyone knows if we are allowed to use other code editors besides Vim for finals? 

This was the instructions sent on luminus:
"The questions will be contained in an encrypted zip file, and you will write your answers in the blank files provided. Ensure that you have the necessary software to open the encrypted zip file, view the pdf file, to edit files, and to view the Java documentation.  **There is no restriction on the software you can use for the above purposes.**"

Thank you!

**Comment by jodytng** [comment]
hi im not sure about this as well, can anyone here help to clarify? thank u!

**Comment by adityajirafe** [comment]
I think based on the announcement, since there's no restriction on the software you can use to edit files, you should be able to use editors like vscode to edit the given .jsh and .java files.

**Comment by rexlionz** [comment]
yup no restrictions would mean that you can use whatever IDE you want! i hope HAHA

**Comment by bryankwe** [comment]
Hi may I ask how to access .jsh files in VSCode? Do we need to install an extension?

**Comment by yangyee-hub** [comment]
I guess we will probably have to wait for further clarification from the TA or Prof.

**Comment by Kathleencsy** [comment]
Can we use notepad to edit the jsh files?

**Comment by rachtan27** [comment]
> Hi may I ask how to access .jsh files in VSCode? Do we need to install an extension?

Hi i just tried to use VSCode and what I did was to copy the files from the zip_mock into a new folder and opened the new folder in VSCode and it seems to work...? 
![image](https://user-images.githubusercontent.com/100123345/165051453-910f79cf-2d60-418c-a3a2-349e5842d5d2.png)


Has anyone else tried accessing the files in VSCode in another or better way?

**Comment by jkjliang** [comment]
![Screenshot (63)](https://user-images.githubusercontent.com/97511958/165056281-bcd2d51d-9e92-4d03-b9d4-10a645adaac4.png)

Yeap it works for me too! I extracted out the zip files into a new folder then dragged the whole folder over to the vscode terminal then I could directly edit the jsh files there. If I'm not wrong I think vscode will automatically prompt you to install some extensions for jsh files if you don't currently have any too!

**Comment by alwinngjw** [comment]
Hi, I think you can just open the  exam folder inside your VS code and start editing on it

Hope this helps

**Comment by Swagston20** [comment]
jus ope the file in vsc

**Comment by calvinseptyanto** [comment]
can try using intellij too!

**Comment by whyyouj** [comment]
just open the folder where you extracted the files and vsc should be able to edit them!
